### PR TITLE
Create ui-c8y-1021-0-0-remove-i-labels-interface-7239.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by DawidSAG

Ticket: [MTM-34821](https://cumulocity.atlassian.net/browse/MTM-34821)
Original PR: [7239](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7239)

[MTM-34821]: https://cumulocity.atlassian.net/browse/MTM-34821?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Remove ILabels interface (#7239)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md]: The Claude AI suggested to use the following title:
```suggestion
title: Remove deprecated ILabels interface
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-34821
+version: 1021.0.0
+---
+Remove ILabels interface (#7239)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md]: The Claude AI suggested to use the following body:
```suggestion
The ILabels interface has been removed. Use ModalLabels interface instead.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md (hunk 2 above)
Comment (rahulc8y) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md]: Seems there is issue with the description. Never heard of the Bosch Iot Things services being used in C8y

**Comment by BeateRixen** [comment]
@DawidSAG Please check this PR. Also see Rahuls comment.

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md [hunk 3]
```diff
@@ -14,4 +14,4 @@ build_artifact:
 ticket: MTM-34821
 version: 1021.0.0
 ---
-Remove ILabels interface (#7239)
\ No newline at end of file
+The ILabels interface has been removed. Use ModalLabels interface instead.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md]:
```suggestion
The deprecated ILabels interface has been removed. Instead, the ModalLabels interface is used.
```

Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md]: Has been removed from wehere? From the Web SDK?

Comment (DawidSAG) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md]: yes, it was removed from the Web SDK

Comment (DawidSAG) [reply content/change-logs/application-enablement/ui-c8y-1021-0-0-remove-i-labels-interface-7239.md]:
```suggestion
The deprecated ILabels interface has been removed from the Web SDK. Instead, the ModalLabels interface is used.
```
?

**Review by DawidSAG — APPROVED** [review]

**Comment by BeateRixen** [comment]
@janhommes @rahultsag Please check this PR

**Review by janhommes — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**